CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Community Code of Conduct
+
+Cilium follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
Even if we follow this code of conduct, we should make it clear how conduct violations should be reported as the upstream document here refers to either conduct@kubernetes.io (for kubernetes, so doesn't apply here), "A CNCF project maintainer" which is probably the closest but doesn't define any method, or the CNCF mediator Mishi Choudhary.

Reply:
I've modified it